proxy/envoy/BUILD

```diff
@@ -59,8 +59,65 @@ genrule(
 )
 
 sh_test(
-    name = "config_test",
+    name = "envoy_config_v0_test",
```

Comment:
Do these need a dependency on `envoy_bin` or does bazel figure that out via `data` attribute?